modules/auxiliary/scanner/http/epmp1000_cmd_exec.rb

```diff
@@ -231,8 +231,6 @@ def do_login(user, pass)
             }
           )
 
-          vprint_line("#{res.body}")
```

Comment:
I think this should be changed to print_line.

Reply:
Same as [this comment](https://github.com/rapid7/metasploit-framework/pull/9316#discussion_r158294341)

Reply:
We need to print `res.body` for this module, as it contains results of command execution. May be use `print_good` instead?